packages/ui/src/components/carousel-item/carouselItem.vue

```diff
@@ -0,0 +1,65 @@
+<script setup lang="ts">
+  import { computed, getCurrentInstance, inject, onMounted, onUnmounted } from 'vue';
+  import { isDefined } from '@vueuse/core';
+  import type {
+    AcvCarouselItemProps,
+    AcvCarouselItemSlots
+  } from './carouselItem.ts';
+
+  import {
+    CAROUSEL_INJECTION_KEY
+  } from '@/components/carousel/carousel.ts';
+  import './carouselItem.css';
+
+  /**
+   * Short description of the CarouselItem component for the generated documentation...
+   *
+   * @component AcvCarouselItem
+   */
+  defineOptions({
+    name: 'AcvCarouselItem'
+  });
+
+  const props = defineProps<AcvCarouselItemProps>();
+
+  // defineEmits<AcvCarouselItemEvents>();
+  defineSlots<AcvCarouselItemSlots>();
+
+  const name = computed(() => props.name || getCurrentInstance()?.uid);
+
+  const carouselInjection = inject(CAROUSEL_INJECTION_KEY);
+
+  if (!carouselInjection) {
+    throw new Error('Carousel must be provided');
+  }
+
```

Comment:
it's so strange that you use this error here, js will throw you the same error

Reply:
No, I checked, the js will output something like: `TypeError: Cannot read properties of undefined (reading 'model')`. 
What's not so clear. It also allows you not to check carouselInjection for null every time